src/api/constants/index.ts

```diff
@@ -143,6 +143,29 @@ export const ACCOUNT = {
   MAX_AUTH_RETRIES_TIMEFRAME: DATE_24_HOURS_IN_THE_PAST(),
 };
 
+export const FINANCIAL_DETAILS = {
+  ENCRYPTION: {
+    CIPHER: {
+      ENCODING: 'hex' as BufferEncoding,
+      STRING_ENCODING: 'base64' as BufferEncoding,
+      ENCRYPTION_METHOD: 'aes-256-cbc',
+      OUTPUT_ENCODING: 'utf-8' as BufferEncoding,
+    },
+    KEY: {
+      ALGORITHM: 'sha512' as Algorithm,
+      SIGNATURE: String(process.env.LOSS_PAYEE_ENCRYPTION_KEY),
+      SUBSTRING_LOWER_VALUE: 0,
+      SUBSTRING_UPPER_VALUE: 32,
+    },
+    IV: {
+      BYTES_SIZE: 16,
+      ENCODING: 'base64' as BufferEncoding,
+      SLICE_LOWER_VALUE: 0,
+      SLICE_UPPER_VALUE: 16,
```

Comment:
Instad of `X_LOWER_VALUE` and `X_UPPER_VALUE` can they be called `X_INDEX_START`, `X_INDEX_END`? For `SUBSTRING` and `SLICE`